v20.2/cockroach-node.md

```diff
@@ -117,18 +117,25 @@ Flag | Description
 `--stats` | Show node disk usage details.
 `--timeout` | Set the duration of time that the subcommand is allowed to run before it returns an error and prints partial information. The timeout is specified with a suffix of `s` for seconds, `m` for minutes, and `h` for hours. If this flag is not set, the subcommand may hang.
 
-The `node decommission` subcommand also supports the following general flag:
+The `node decommission` subcommand also supports the following general flags:
 
 Flag | Description
 -----|------------
 `--wait` | When to return to the client. Possible values: `all`, `none`.<br><br>If `all`, the command returns to the client only after all replicas on all specified nodes have been transferred to other nodes. If any specified nodes are offline, the command will not return to the client until those nodes are back online.<br><br>If `none`, the command does not wait for the decommissioning process to complete; it returns to the client after starting the decommissioning process on all specified nodes that are online. Any specified nodes that are offline will automatically be marked as decommissioning; if they come back online, the cluster will recognize this status and will not rebalance data to the nodes.<br><br>**Default:** `all`
+`--self` | Applies the operation to the node against which the command was run (e.g., via `--host`).
```

Comment:
Do we want to feature this flag in any of the instructions, or is that not necessary?

Reply:
Good point. I'm not sure it will make a big impact on how we handle the orchestration steps (will double-check), but it will simplify parts of the node decommissioning doc. I will update in this PR.